python/ngen_conf/src/ngen/config/cfe.py

```diff
@@ -5,15 +5,65 @@
 
 from .bmi_formulation import BMIC
 
+
 class CFEParams(BaseModel):
-    """Class for validating CFE Parameters
-    """
-    maxsmc: Optional[float]
+    """Class for validating CFE Parameters"""
+
+    b: Optional[float]
+    """Pore size distribution index"""
     satdk: Optional[float]
+    """Saturated hydraulic conductivity"""
+    satpsi: Optional[float]
+    """saturated capillary head"""
     slope: Optional[float]
-    bb: Optional[float]
-    multiplier: Optional[float]
+    """Linear scaling of "openness" of bottom drainage boundary"""
+    maxsmc: Optional[float]
+    """saturated soil moisture content"""
+    wltsmc: Optional[float]
+    """wilting point soil moisture content"""
+    refkdt: Optional[float]
+    """
+    Surface runoff parameter; REFKDT is a tuneable parameter that significantly
+    impacts surface infiltration and hence the partitioning of total runoff into
+    surface and subsurface runoff. Increasing REFKDT decreases surface runoff
+    """
```

Comment:
Probably worth noting in the docstring here that this is only used with Schaake option
```suggestion
    """
    Surface runoff parameter; REFKDT is a tuneable parameter that significantly
    impacts surface infiltration and hence the partitioning of total runoff into
    surface and subsurface runoff. Increasing REFKDT decreases surface runoff.
    Only applicable when surface_water_partitioning_scheme is Schaake.
    """
```

Reply:
These are great suggestions! Thanks for the careful eyes! I pulled straight from `cfe`'s confirmation docs. I will make your suggested changes (tomorrow, AFK at the moment) and open a pr to cfe to improve their docs (w/ co-author of course).